src/components/home/modals/CancelLease.vue

```diff
@@ -178,6 +188,15 @@ export default {
             this.coldSignature = ''
             this.hasConfirmed = false
         },
+        coldApi: function() {
+            let a = JSON.parse(window.localStorage.getItem(this.defaultAddress)).coldAddresses
+            let tempApi = JSON.parse(a)[this.fromAddress].api
+            if (tempApi === 1 && BigNumber(this.amount).isLessThan(BigNumber(Number.MAX_SAFE_INTEGER).dividedBy(1e8))) {
+                return 1
```

Comment:
L194-L195
Magic number 1 in '''tempApi === 1 ''' and '''return 1''' should be declared as constant variable.

Reply:
This `1` means API version 1. If declare as constant, how to name it?